script.js

```diff
@@ -1,111 +1,76 @@
-const inputTitle = document.querySelector('#title');
-const inputAuthor = document.querySelector('#author');
-const addBtn = document.querySelector('.add-but');
+// eslint-disable-next-line max-classes-per-file
 const booksSection = document.querySelector('.booksSection');
-
-let books = [];
-
-const renderBooks = () => {
-  booksSection.innerHTML = '';
-  books.forEach((book) => {
-    const newBook = document.createElement('div');
-    newBook.classList.add('book');
-    newBook.innerHTML = `
-      <p>${book.title} by ${book.author}</p>
-      <button class="remove-but">Remove</button>
-    `;
-    booksSection.appendChild(newBook);
-    localStorage.setItem('books', JSON.stringify(books));
- });
-};
-
-window.addEventListener('DOMContentLoaded', () => {
-  const storageBooks = JSON.parse(localStorage.getItem('books'));
-  if (storageBooks) {
-    books = storageBooks;
-    renderBooks();
+class First {
```

Comment:
[OPTIONAL] Please remember to give a meaningful name to your classes. It makes your code clearer and easier to understand for other developers. :nerd_face: :nerd_face: